django_geosource/models.py

```diff
@@ -373,3 +374,53 @@ def refresh_data(self):
 
     def _get_records(self, limit=None):
         return []
+
+
+class CSVSource(Source):
+    COMA = ("coma", ",")
+    SEMI_COLON = ("semicolon", ";")
+    TAB = ("tabulation", "\t")
+    SPACE = ("space", " ")
+    COLUMN = ("column", " ")
+    QUOTATION_MARK = ("quotationmark", '"')
+    POINT = ("point", ".")
+
+    SEPARATORS = (COMA, SEMI_COLON, TAB, SPACE, COLUMN)
+    DECIMAL_SEPARATORS = (COMA, SEMI_COLON, POINT, SPACE)
+    DELIMITERS = (QUOTATION_MARK,)
+
+    file = models.FileField(upload_to="geosource/csv/%Y")
+    encoding_field = models.CharField(max_length=100)
```

Comment:
encoding

Reply:
Field has been removed, this data is now stored in the `settings` field